frontend/packages/topology/src/filters/KindFilterDropdown.tsx

```diff
@@ -36,6 +37,13 @@ const KindFilterDropdown: React.FC<KindFilterDropdownProps> = ({
         type: TopologyDisplayFilterType.kind,
         id: kind,
         label: model ? model.label : kind,
+        labelKey: model
+          ? model?.labelKey
+            ? t(model?.labelKey)
+            : model.label
+          : translationForResourceKind[kind]
+          ? t(translationForResourceKind[kind])
+          : kind,
```

Comment:
We use `labelKey` for a i18n string that needs to be used with `t()`.

So I recommend to keep "label" for the translated label here.

But it makes sense to now use it for the data-test attribute in the `SelectOption`. For this, you use the old logic if needed:

```suggestion
        "data-test": model ? model.label : kind,
        label: model
          ? model?.labelKey
            ? t(model?.labelKey)
            : model.label
          : translationForResourceKind[kind]
          ? t(translationForResourceKind[kind])
          : kind,
```